test/unit/tracked_request_test.rb

```diff
@@ -10,19 +10,6 @@ def test_marshal_dump_load
     loaded = Marshal.load(dumped)
     assert_false loaded.nil?
   end
-
-  def test_restore_store
-    faux_store = ScoutApm::FakeStore.new
-    context = ScoutApm::AgentContext.new
-    tr = ScoutApm::TrackedRequest.new(context, faux_store)
-    assert_equal faux_store, tr.instance_variable_get("@store")
-
-    tr.prepare_to_dump!
-    assert_nil tr.instance_variable_get("@store")
-
-    tr.restore_store
-    assert_equal context.store, tr.instance_variable_get("@store")
-  end
```

Comment:
This test file wasn't running because it was named `test_...` instead of `..._test.rb`. When I fixed that, this test failed because it's calling a method that no longer exists.